warehouse/query-core/src/main/java/datawave/query/planner/rules/RegexDotallTransformRule.java

```diff
@@ -0,0 +1,61 @@
+package datawave.query.planner.rules;
+
+import com.google.common.collect.Multimap;
+import datawave.query.config.ShardQueryConfiguration;
+import datawave.query.jexl.JexlASTHelper;
+import datawave.query.jexl.functions.EvaluationPhaseFilterFunctions;
+import datawave.query.jexl.functions.EvaluationPhaseFilterFunctionsDescriptor;
+import datawave.query.jexl.functions.FunctionJexlNodeVisitor;
+import datawave.query.jexl.functions.JexlFunctionArgumentDescriptorFactory;
+import datawave.query.jexl.functions.arguments.JexlArgumentDescriptor;
+import datawave.query.postprocessing.tf.Function;
```

Comment:
Looks like this might be an unused import.

Reply:
thank you